pkg/node/bootstrap.go

```diff
@@ -371,3 +369,20 @@ func batchStoreExists(s storage.StateStorer) (bool, error) {
 
 	return hasOne, err
 }
+
+const OverlayNonce = "overlayV2_nonce"
+
+func overlayNonceExists(s storage.StateStorer) ([]byte, bool, error) {
```

Comment:
these functions do not belong to this file

Reply:
where would be a more fitting place?

Reply:
in `noode/node.go` or `node/statestore.go` ?